### ISSUE TITLE
Train with keras-2

### ISSUE THREAD

**Issue by gkhan496** [uw-biomedical-ml/irf-segmenter#9]
Hi, I have a database of this kind. I want to train in the same way(for segmetation). Is there a source you can suggest? Finally, Do you plan to update the application for keras2 or higher versions of tensorflow ? 

Thank you.

**Comment by ayl** [comment]
Thanks for your comment.  This is the first request we have had to update the application for keras2. 

We may consider rereleasing this by retraining for keras2 and releasing the training code that we used. Would that be of interest?

**Comment by gkhan496** [comment]
Exactly. I think you can train with a different model and compare the results with unet. The results look great, but the version is too old. I'd like to help if you reference me. Good work. 

Thank you for answer.
